packages/qdrant-loader/src/qdrant_loader/cli/serve/http.py

```
@@ -0,0 +1,358 @@
+        graph_name = graph_name or graph_cfg.graph_name
+
+    try:
+        # Lazy import so optional backend deps don't break module import
+        from qdrant_loader_core.graph.falkor_store import FalkorGraphStore
+
+        return FalkorGraphStore(
+            host=host or "localhost",
+            port=int(port) if port else 6379,
+            graph_name=graph_name or "default_graph",
+        )
+    except Exception as exc:  # pragma: no cover - surface import/runtime errors
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)